### PR TITLE
Remove composer.lock from git export

### PR DESCRIPTION
**Pull request by glensc**
This is to avoid false-positive as being sub-project with `snyk monitor`.

This makes composer package contain only needed files:
- readme, license
- php source

### CODE REVIEW

**Review by deguif — APPROVED** [review]

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.travis.yml export-ignore
+/composer.lock export-ignore
+/docs export-ignore
```
Comment (deguif) [inline .gitattributes:5]: Sure about this one?

Comment (glensc) [reply .gitattributes:5]: yes. I am:

> This makes composer package contain only needed files:
> 
> - readme, license
> - php source

you can always install --prefer-source to get all files, or browse online rest of the files. documentation in vendor did is not required for package to be used.

**Comment by glensc** [comment]
@preetishishodia7 seems you totally missed the context. 

you have committed your project composer.lock to the project:
- https://github.com/sendinblue/APIv3-php-library/blob/11af6be3c0a37f8d086238681256f796752dd1ad/composer.lock

now if I install your library, it will be present in vendor/sendinblue/api-v3-sdk/composer.lock

now security scanner like snyk, sees the file and thinks it's another project to scan.

this pr was adding the file to .gitattributes ignore, so when composer is installed with --prefe-dist, the filesendinblue/api-v3-sdk/composer.lock file will not be added to the .zip that composer downloads.

besides, it's common practice to exclude files not relevant for using your project using .gitattributes, these include tests, and extra development files.

**Comment by preetishishodia7** [comment]
HI @glensc thanks for explaining, will look into it.

**Comment by glensc** [comment]
some random links from internet on the topic to get your started:
- https://github.com/humanmade/Cavalcade/issues/97
- https://stackoverflow.com/questions/16528584/ignore-a-git-directory-in-composer-package
- https://stackoverflow.com/questions/74631768/composer-exclude-files-from-package
- https://medium.com/@pavlakis/sharing-with-composer-and-gitattributes-4423f5c4967f

**Pull request closed without merging**